docs/resources/rf_stack.md

```diff
@@ -0,0 +1,194 @@
+---
+subcategory: "Application Orchestration Service (AOS)"
+---
+
+# huaweicloud_rf_stack
+
+Manage an RF resource stack and deployment resources through this resource.
```

Comment:
This seems a bit difficult for understading, how about using something short like "Provides an RF resource stack."

Reply:
sounds great.